src/test_utils/hive/mod.rs

```diff
@@ -98,10 +97,6 @@ impl HiveGenesisConfig {
                     ),
                 ]);
 
-                let key = get_storage_var_address("ERC20_allowances", &[starknet_address, kakarot_address])?;
-                fee_token_storage.insert(key, u128::MAX.into());
-                fee_token_storage.insert(key + Felt::ONE, u128::MAX.into());
-
```

Comment:
why is this removed?

Reply:
Shouldn't be removed, was working on it, just put it back